mimic/session.py

```diff
@@ -102,6 +112,7 @@ def _new_session(self, username_key=None, **attributes):
         self._username_to_token[username_key] = session.token
         self._token_to_session[session.token] = session
         self._tenant_to_token[session.tenant_id] = session.token
+        self._user_id_to_sesion[session.user_id] = session
```

Comment:
`sesion`→`session`

The pattern for most of the other things here though is that we have one thing that points at the actual `session` object, `_token_to_session`, and then lots of other dictionaries which point at the token, thereby indirectly referencing the session.  I'd kinda like to keep it to one direct reference, so that we can clear out the session by popping it out of one place instead of N places.